masterlist.yaml

```
@@ -8061,6 +8061,25 @@ plugins:
       - name: 'xulArriusCreek.esp'
         condition: 'not version("Unique Landscapes.esp", "2.0.0", >=)'
 
+  - name: '(Better Abandoned House.*|Cheydinhal Land Texture Fix.*)?\.esp'
```

It would apply to every plugin that exist if you did it that way.

```suggestion
  - name: '(Better Abandoned House|Cheydinhal Land Texture Fix).*\.esp'
```